charts/horizon/values.yaml

```diff
@@ -62,7 +62,7 @@ ingest:
 
   ## This variable allows overrides of default settings
   horizonConfig:
-    {}
+    stellarCoreUrl: http://reynholm-industries-core:11626
```

Comment:
This should be overriden in the `values-reynholmindustries.yaml` file not in the horizon chart's values.